### PR TITLE
Start to make changelog for ipywidgets 8 more friendly to users

### PR DESCRIPTION
**Pull request by jasongrout**
This starts to make the changelog more prose instead of just bullet points.

### CODE REVIEW

**Comment by jasongrout** [comment]
@ibdafna - any suggestions for this changelog? You did the highlights tour for the tutorial.

**Comment by ibdafna** [comment]
@jasongrout I glanced over it and it seems to cover everything I can think of. It's probably a good start!

File: docs/source/changelog.md [hunk 1]
```diff
@@ -10,21 +10,65 @@ See also the
...
+
+#### Date and time pickers
+
+A new date/time and time picker has been added
+
+- Add datetime and time pickers ([#2715](https://github.com/jupyter-widgets/ipywidgets/pull/2715))
+
+#### Collaboration
+
+By default, ipywidgets 8 enables a collaboration mode, where widget updates from one frontend are reflected back to other frontends, enabling a consistent state between multiple users and fixing synchronization race conditions for a more robust synchronization between kernel and frontend. To disable echo update messages across ipywidgets, set the environment variable `JUPYTER_WIDGETS_ECHO` to `0`. To opt a specific attribute of custom widget out of echo updates, tag the attribute with `echo_update=False` metadata (we do this in core for the FileUpload widget's `data` attribute). [#3195](https://github.com/jupyter-widgets/ipywidgets/pull/3195), [#3343](https://github.com/jupyter-widgets/ipywidgets/pull/3343), [#3394](https://github.com/jupyter-widgets/ipywidgets/pull/3394), [#3407](https://github.com/jupyter-widgets/ipywidgets/pull/3407)
```
Comment (williamstein) [inline docs/source/changelog.md:52]: Please add a remark about performance/bandwidth implications.

Comment (williamstein) [reply docs/source/changelog.md:52]: Why would you want to disable?

**Comment by vidartf** [comment]
Let's also have the changelog link to the spec diff for those that are interested: https://github.com/jupyter-widgets/ipywidgets/issues/3455

**Comment by vidartf** [comment]
Closes #3455.

**Pull request merged**